docs/build_stacks_rst.py

```diff
@@ -107,8 +107,31 @@ def conda_list(self, image):
             'conda', 'list', '--json']
         joined_args = " ".join(args)
         out = subprocess.check_output(joined_args, shell=True)
-        out = conda_json_to_version_dict(out)
-        return out
+        versions = conda_json_to_version_dict(out)
+        return versions
+
+    def get_conda_history(self, image):
+
+        # return "docker run ... conda list"
+        args = [
+            'docker', 'run', '-it',
+            '--entrypoint=""',
+            image,
+            'conda', 'env', 'export', '--from-history', '--json']
```

Comment:
this seems to require a pretty new version of conda. I don't think this works on repo2docker built images yet.